jerry-core/ecma/base/ecma-alloc.h

```diff
@@ -30,108 +30,108 @@
  *
  * @return pointer to allocated memory
  */
-extern ecma_object_t *ecma_alloc_object (void);
+extern ecma_object_t * ecma_alloc_object (void);
```

Comment:
Now I'm a bit lost. What is the current style for such cases? We need a space after the `*` character when using it as a return type? We should really have a style guide somewhere written down.